### PR TITLE
Feature/table

### PR DESCRIPTION
**Pull request by ndalonzo**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -28,9 +28,15 @@
     "styled-components": "^4.1.0"
   },
   "dependencies": {
+    "@types/react-modal": "^3.8.0",
```
Comment (ben-dalton) [inline package.json]: The types should go in `devDependencies`. `dependencies` get wrapped up in the shipped bundle, so third party libraries that we don't want the end user to have to install separately can go here.

Comment (ndalonzo) [reply package.json]: Updated!  I also extracted the TablePagination into its own component.

File: package.json [hunk 2]
```diff
@@ -28,9 +28,14 @@
     "styled-components": "^4.1.0"
   },
   "dependencies": {
+    "@types/react-slick": "^0.23.2",
```
Comment (ben-dalton) [inline package.json]: Can you move the rest of the types packages to `devDependencies` too?

File: package.json (hunk 1 above)
Comment (ndalonzo) [reply package.json]: Sorry, should have seen that!  Updated.

File: package.json (hunk 2 above)
Comment (ndalonzo) [reply package.json]: Sorry, should have seen that! Updated.

**Pull request merged**